cmd/util/cmd/exec-data-json-export/result_exporter.go

```diff
@@ -0,0 +1,87 @@
+package jsonexporter
+
+import (
+	"bufio"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/onflow/flow-go/cmd/util/cmd/common"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module/metrics"
+	"github.com/onflow/flow-go/storage/badger"
+)
+
+type result struct {
+	ResultID         string   `json:"result_id"`
+	PreviousResultID string   `json:"prev_result_id"`
+	BlockID          string   `json:"block_id"`
+	FinalStateCommit string   `json:"state_commitment"`
+	Chunks           []string `json:"chunks"`
+}
+
+// ExportResults exports results
+func ExportResults(blockID flow.Identifier, dbPath string, outputPath string) error {
+
+	// traverse backward from the given block (parent block) and fetch by blockHash
+	db := common.InitStorage(dbPath)
+	defer db.Close()
+
+	cacheMetrics := &metrics.NoopCollector{}
+	headers := badger.NewHeaders(cacheMetrics, db)
+	results := badger.NewExecutionResults(db)
+	activeBlockID := blockID
+
+	outputFile := filepath.Join(outputPath, "results.jsonl")
```

Comment:
```suggestion
	outputFile := filepath.Join(outputPath, "results.json")
```

Reply:
thats actually jsonl file and not json, it will be a json per line.